### PR TITLE
Update itsxpress to 2.0.0

### PR DESCRIPTION
**Pull request by seina001**
Updating itsxpress to 2.0.0 with source now from GitHub release and new dependency versions for newer Qiime2 compatibility. See changelog in itsxpress repo. 

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by seina001** [comment]
@BiocondaBot please merge

**Comment by seina001** [comment]
@BiocondaBot please add label

**Comment by seina001** [comment]
@BiocondaBot please fetch artifacts

File: recipes/itsxpress/meta.yaml [hunk 1]
```diff
@@ -22,14 +22,14 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.8
+    - python =3.8.16
```
Comment (bgruening) [inline recipes/itsxpress/meta.yaml:25]: pinning  a recipe this strict is not a good idea. Why is python 3.9 not supported? or 3.10?

Comment (seina001) [reply recipes/itsxpress/meta.yaml:25]: I agree, this is unnecessary. I'll change it back to >=3.8 shortly. This was originally to help with dependency issues with Qiime2, as this package can also be used as a plugin for Qiime but those issues have been fixed.

**Comment by seina001** [comment]
@BiocondaBot please fetch artifacts

File: recipes/itsxpress/meta.yaml [hunk 2]
```diff
@@ -28,8 +28,8 @@ requirements:
     - python >=3.8
     - biopython >=1.79
     - bioconda::hmmer =3.1b2
-    - bioconda::bbmap =38.96
-    - bioconda::vsearch =2.21.1
+    - bioconda::vsearch =2.22.1
```
Comment (bgruening) [inline recipes/itsxpress/meta.yaml:31]: Please remove the bioconda: this is. It needed.

**Review by bgruening — COMMENTED** [review]
Can you please use an SPDX license identifier, if available?

**Review by shenwei356 — APPROVED** [review]

**Pull request merged**